controllers/components/kueue/kueue_support.go

```diff
@@ -33,3 +36,16 @@ func manifestsPath() odhtypes.ManifestInfo {
 		SourcePath: "rhoai",
 	}
 }
+
+func extramanifestsPath() odhtypes.ManifestInfo {
+	return odhtypes.ManifestInfo{
+		Path:       odhdeploy.DefaultManifestPath,
+		ContextDir: ComponentName,
+		SourcePath: "rhoai/ocp-4.17-addons",
+	}
+}
+
+// Add OCP 4.17+ specific manifests if Minor > 16 .
+func vapPredicate(context.Context, *odhtypes.ReconciliationRequest) bool {
+	return cluster.GetClusterInfo().Version.Minor > 16
```

Comment:
to make things a little but more clear, maybe we should have a constant for the minimum version and use` SemVer.GTE`

```go
var V4_17 = semver.MustParse("4.17")

func vapPredicate(context.Context, *odhtypes.ReconciliationRequest) bool {
  return cluster.GetClusterInfo().Version.GTE(V4_17)
}
```